src/lib/containers/UserSearchBoxV2.tsx

```diff
@@ -108,6 +110,13 @@ const UserSearchBoxV2: React.FC<UserSearchBoxProps> = props => {
     typeFilter,
   )
 
+  const selectRef = React.useRef<any>(null)
+  React.useEffect(() => {
+    if (focusOnSelect) {
+      selectRef.current && selectRef.current.focus()
+    }
+  })
```

Comment:
Does it still work with this dependency array? I think you might have issues if you try to set the focus on every render.
```suggestion
  }, [focusOnSelect])
```

Reply:
No it doesn't work with the dependency array.